src/axom/inlet/Table.cpp

```diff
@@ -680,27 +675,43 @@ Table& Table::required(bool isRequired)
 
 bool Table::isRequired() const
 {
+  if(isGenericContainer())
+  {
+    bool result = false;
+    forEachContainerElement([&result](Table& table) {
+      if(table.isRequired()) result = true;
```

Comment:
Single line if's are not encouraged in Axom due to the bugs that can be introduced due to them.  Please add curly braces.